frontend/src/components/AddNewInput.tsx

```diff
@@ -0,0 +1,60 @@
+import { Button, FormLabel } from "@chakra-ui/react";
+import React, { useState } from "react";
+
+import InputField from "./InputField";
+
+type AddNewInputProps = {
+  label: string;
+  name: string;
+  placeholder: string;
+  required: boolean;
+};
+
+const AddNewInput = ({
+  label,
+  name,
+  placeholder,
+  required,
+}: AddNewInputProps): React.ReactElement => {
+  const [inputFields, setInputFields] = useState<string[]>([""]);
+
+  const handleInputChange = (
+    e: React.ChangeEvent<HTMLInputElement>,
+    index: number,
+  ) => {
+    const list = [...inputFields];
+    list[index] = e.target.value;
+    setInputFields(list);
+  };
+
+  const handleRemoveField = (index: number) => {
+    setInputFields(inputFields.filter((_, i) => i !== index));
+  };
+
+  const handleAddField = () => {
+    setInputFields([...inputFields, ""]);
+  };
+
+  return (
+    <>
```

Comment:
Can you add in a prop that checks if the field is required or not and add the asterisk accordingly like this:
![image](https://user-images.githubusercontent.com/59517368/146391972-d88e825b-56c8-420e-988d-3027a49e6c8d.png)

It shows how to do that here: https://chakra-ui.com/docs/form/form-control